### ISSUE TITLE
Console window remains open after file type association / flag bug

### ISSUE THREAD

**Issue by Crossed-Omega** [Beep6581/RawTherapee#7095]
Windows 10 Version 10.0.19045 Build 19045, RawTherapee 5.10. Clean install from RawTherapee_5.10_win64.exe. 

## Problem
I have noticed some weird behavior regarding the console window that is opened during the use of the RT GUI. I have opened [a thread on the forum](https://discuss.pixls.us/t/rt-as-default-app-on-w10-opens-a-console-window/43857) about this. My situation is as follows:
- Upon executing rawtherapee.exe from the main directory or using start, the GUI is opened and RT works as intended. No console is opened.
- When associating a file type, such as .rwl to Rawtherapee via either the main settings or "open with" context menu, the respective file may be opened by double clicking on it in File Explorer. A console is opened, after which the GUI is opened. The image can be edited in remote mode and the sidecar is saved correctly. However, upon closing the GUI, "Press any key to exit RawTherapee" is printed to the console. The console does not contain any information from editing operations. 

## Expected behavior
According to the wiki:
> If you pass parameters to the RawTherapee executable it spawns a console window so that you can see the verbose output of your processing. Normally Windows closes this console directly after RawTherapee is terminated. To let you see the output we added a prompt which waits for you to hit a key before closing the console.

Since opening a file with its associated program passes the filepath to RT, it opens the console window which is "expected". It does not automatically close the console window after the GUI is closed. I would expect it to close, especially if nothing is printed to the console at all.

## Quick hacky fix
I have fixed this behavior by changing the registry key `Computer\HKEY_CURRENT_USER\SOFTWARE\Classes\Applications\rawtherapee.exe\shell\open\command` to
`“C:\Program Files\RawTherapee\5.10\rawtherapee.exe” -w “%1”` which successfully suppresses the console. 

Alternatively, as another user in the thread mentioned above states, changing only the specific file association with the key `Computer\HKEY_CURRENT_USER\SOFTWARE\Classes\.rwl` should also work.

I think there are some issues regarding this approach. It is nonsensical that the only way to change the behavior described above can be changed only by fiddling with registry keys. An average user should not have to do this. **If possible, could a settings header be added with checkmarks for the behavior on normal and associated file startup?** Even when `verbose=false` in the options and nothing is printed, having to press a key to close a window without any information is not very useful.

## Minor flag bug
A minor bug, which is likely incorrect behavior, is that:
- `“C:\Program Files\RawTherapee\5.10\rawtherapee.exe” “%1”` leads to a persistent console and a remote GUI (correct);
- `“C:\Program Files\RawTherapee\5.10\rawtherapee.exe” -w “%1”` leads to no console and a remote GUI (correct);
- `“C:\Program Files\RawTherapee\5.10\rawtherapee.exe” -R “%1”` leads to no console and a non-remote GUI whilst this should, according to the wiki, lead to a non-remote GUI AND a persistent console (incorrect);
- `“C:\Program Files\RawTherapee\5.10\rawtherapee.exe” -w -R “%1”` leads to no console and a non-remote GUI (correct).
In short, **could the flag behavior be looked over and applied correctly?**

I hope I made this issue clear. Feedback is appreciated, thanks in advance.

**Comment by Lawrence37** [comment]
The `-w` flag needs to be added in the Inno setup configuration. It doesn't look like t was intentionally left out.
